src/lib/components/copyable.svelte

```diff
@@ -11,20 +10,37 @@
   const { copy, copied } = copyToClipboard(500);
 </script>
 
-<div
-  class="group flex items-center gap-2 {$$props['container-class']}"
-  on:click={clickAllToCopy ? (e) => copy(e, content) : noop}
->
-  <slot>
-    <span class={$$props.class} class:select-all={!$$slots.default}
-      >{content}</span
-    >
-  </slot>
-  <button on:click={(e) => copy(e, content)}>
-    <Icon
-      name={$copied ? 'checkmark' : 'copy'}
-      stroke={color}
-      class={`${visible ? 'visible' : 'invisible group-hover:visible'}`}
-    />
+{#if clickAllToCopy}
+  <button
```

Comment:
Might there be a way we could conditionally add an `on:click`? 

If so, instead of this conditional with repeated code, we could so something like `<svelete:element this={clickAllToCopy ? 'button' : 'div'} />` ✨

Reply:
We should do that… but maybe as a different PR?